PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -284,7 +286,7 @@ struct HfTreeCreatorToXiPi {
     // Filling candidate properties
     rowCandidateFull.reserve(candidates.size());
     for (const auto& candidate : candidates) {
-      fillCandidate(candidate, -7, -7);
+      fillCandidate(candidate, -7, -7, -7);
```

Comment:
Why `-7`? Shouldn't it be better to fill it with `RecoDecay::OriginType::None`?

Reply:
Hi @fgrosa, thanks for the comments! Sure, I change it